### ISSUE TITLE
Improve outer menu

### ISSUE THREAD

**Issue by felipebiox** [OmniDB/OmniDB#1180]
Outer Menu is not very practical yet.

As of now, the outer menu is collapsed by default when working on a clientWindow that is smaller than 1981px.
While extended, it is floating above the content.

This is not ideal, forcing the user to navigate the menu or expand it manually to better locate connections.

Suggestion is to apply the large screen layout when extended and apply it as the default status.

![menu_change](https://user-images.githubusercontent.com/11302800/97593840-d57c6480-19e0-11eb-9c42-07d2120195fd.png)

**Comment by Leonardofreua** [comment]
I believe it would be interesting to have the possibility to collapse the **sidebar of the selected DB**. So it would make it possible to view the editor broader perspective (I speak from experience).

**Comment by felipebiox** [comment]
Hi @Leonardofreua ,
By sidebar you refer to the area of database tree + properties/ddl?

So, a toggler quick-action to collapse it entirely?

**Comment by Leonardofreua** [comment]
> Hi @Leonardofreua ,
> By sidebar you refer to the area of database tree + properties/ddl?
> 
> So, a toggler quick-action to collapse it entirely?

Exactly. Today have a feature to drag the sidebar, but it would be interesting to collapse it.

**Comment by felipebiox** [comment]
This implementation contemplates a few structural changes to the layout besides the toggling action.

- Removing the fixed style of the menu.
- The main container of the primary tab is now styled with css GRID with template set in place: `'menu content' 100% / auto minmax(0, 1fr)`. With this, the outer_menu still allows internal scroll content when heavily populated, while the content area can expand/shrink.
- The default status of the outer_menu is expanded, but the user can collapse it.
- Collapsing/Expanding the menu changes the space available for the content, and many components are not fluid. This means that the toggling action also triggers a resize on these elements, defaulting to the context available. This has been centralized in a refreshHeights method that calls a v_tab_tag.resize() method. Each type of inner_tab has this resize function callback assigned to them.
- Floating panels, such as the snippets global panel and the full-view panel have been re-worked to account for the new structure as well.
- Added a toggle to the main workspace that collapses/expands the content from the left (database tree + properties/ddl). This also triggers resizes for components.

![expanded_menu](https://user-images.githubusercontent.com/11302800/97926631-38079480-1d42-11eb-9b47-2aa2f5690aea.png)

![collapsed_menu](https://user-images.githubusercontent.com/11302800/97926646-3d64df00-1d42-11eb-84f6-e1e159f876fb.png)

![collapsed_div](https://user-images.githubusercontent.com/11302800/97926659-40f86600-1d42-11eb-9eb6-723f258165fa.png)

![full-view](https://user-images.githubusercontent.com/11302800/97926670-45bd1a00-1d42-11eb-9363-ae61c0bda6ed.png)

![snippets](https://user-images.githubusercontent.com/11302800/97926676-49e93780-1d42-11eb-8f4e-968135e18b23.png)
